### PR TITLE
Update registration and publishing documentation

### PR DESCRIPTION
**Pull request by gebansal**

### CODE REVIEW

File: microsoft-edge/extensions-chromium/publish/publish-extension.md [hunk 1]
```diff
@@ -83,7 +83,12 @@ On the **Properties page**, enter the following information to specify propertie
 | Support contact details | The URL to your support webpage, or the email address to contact your support team.  |  
 | Mature content | Checkbox to specify if your extension includes mature content.  Extension rating helps determine the appropriate age group of the target audience of your extension.  To help determine if your extension has mature content, go to [Microsoft Edge add-ons catalog developer policies][MicrosoftEdgeAddonsCatalogDeveloperPolicies].  |  
 
-Select **Save** to continue to the **Store listings** section.  
+
+Select **Save** to continue to the **Store listings** section.
+
+> [!Important]
+> Your developer/organization name, website URL and the support contact details are displayed to the users on the Edge add-ons store.
```
Comment (Reezaali) [inline microsoft-edge/extensions-chromium/publish/publish-extension.md:90]: Does the name come from the manifest file? Can we include that clarification?
 
Your developer/organization name included in the manifest file, website URL...

**Review by Reezaali — CHANGES_REQUESTED** [review]

File: microsoft-edge/extensions-chromium/publish/publish-extension.md (hunk 1 above)
Comment (gebansal) [reply microsoft-edge/extensions-chromium/publish/publish-extension.md]: It comes from account details which are submitted at the time of developer registration.

**Comment by gebansal** [comment]
Registration documentation Acrolinx score - 91
Publishing documentation Acrolinx score - 94

File: microsoft-edge/extensions-chromium/publish/publish-extension.md (hunk 1 above)
Comment (Reezaali) [reply microsoft-edge/extensions-chromium/publish/publish-extension.md]: Can we update to:

> Your developer/organization name, website URL, and support contact details that were submitted during registration are displayed to users on the Microsoft Edge add-ons store.

Comment (gebansal) [reply microsoft-edge/extensions-chromium/publish/publish-extension.md]: done

File: microsoft-edge/extensions-chromium/publish/create-dev-account.md [hunk 2]
```diff
@@ -3,30 +3,37 @@ description: Learn how to register to a developer account to publish extensions
...
+> Today the Microsoft Edge extensions does not support signing up with a work or school account.  In the future, the Microsoft Edge extensions team plans to support linking Azure AD tenants to MSA accounts for extension management.  
 
 ## Enroll in the Microsoft Edge program on Partner Center  
 
-1.  Go to the [developer page][MicrosoftPartnerCenter] and select **GO TO DASHBOARD**.  
-1.  If you are not already signed in with a **Microsoft account**, sign-in now, or create a new Microsoft account.  You use the same Microsoft account that you use to sign into your developer account.  After you sign-in, this registration form is displayed.  
+1.  Navigate to the [developer page][MicrosoftPartnerCenter] and choose **GO TO DASHBOARD**.  
+1.  If you have a Microsoft account, sign into it now.  If not, create a new Microsoft account.  Use the same Microsoft account that you use to sign into your developer account.  After you sign into your developer account, the following registration form is displayed.
```
Comment (Reezaali) [inline microsoft-edge/extensions-chromium/publish/create-dev-account.md:33]:
```suggestion
1.  Navigate to the [developer page][MicrosoftPartnerCenter] and choose **GO TO DASHBOARD**.  
1.  If you have a Microsoft account, sign into it now.  If not, create a new Microsoft account.  Use the same Microsoft account that you use to sign into your developer account.  After you sign in, a registration form is displayed. The following table describes the fields on the registration form.  
```

File: microsoft-edge/extensions-chromium/publish/create-dev-account.md [hunk 3]
```diff
@@ -53,79 +61,46 @@ If you do not have an account or if you have an existing commercial account with
...
-1. To complete your enrollment, select **Finish**.  
-
+    
+1.  Before submitting your registration form, read and accept the terms and conditions of the [Microsoft Edge Developer Agreement][MicrosoftAppDeveloperAgreement].  
+1.  To complete your enrollment, choose **Finish**.  
+    
 ## Next steps  
 
-You may view your verification status on the **Account settings** page in Partner Center.  While you wait for the verification process to complete; you may continue to build, test, and prepare your submissions.  
+To display your verification status, navigate to Partner Center > **Account settings**.  While you wait for the verification process to complete; continue to build, test, and prepare your submissions.
```
Comment (Reezaali) [inline microsoft-edge/extensions-chromium/publish/create-dev-account.md:101]:
```suggestion
To display your verification status, navigate to Partner Center > **Account settings**.  While you wait for the verification process to complete, continue to build, test, and prepare your submissions.  
```

**Review by Reezaali — CHANGES_REQUESTED** [review]

File: microsoft-edge/extensions-chromium/publish/publish-extension.md [hunk 4]
```diff
@@ -12,13 +12,15 @@ keywords: edge-chromium, extensions development, browser extensions, addons, par
 
 After you develop and test your extension, you're ready to distribute your extension. Use the Microsoft Edge add-ons catalog to distribute your extension.  To release your existing Chromium extension for Microsoft Edge users, navigate to [port your existing Chromium extension][PortChromiumExtension].  
 
-Publish your extension to the Microsoft Edge add-ons catalog to increase the reach of it and make it available to users of Microsoft Edge.  This article provides the process to submit your extension to the Microsoft Edge add-ons catalog.  
+Publish your extension to the Microsoft Edge add-ons catalog to increase the reach of it and make it available to users of Microsoft Edge.  
+    
+This article provides the process to submit your extension to the Microsoft Edge add-ons catalog.
```
Comment (Reezaali) [inline microsoft-edge/extensions-chromium/publish/publish-extension.md:17]:
```suggestion
Publish your extension to the Microsoft Edge add-ons catalog to increase the reach of it and make it available to users of Microsoft Edge.  This article provides the process to submit your extension to the Microsoft Edge add-ons catalog.  
```

**Review by Reezaali — APPROVED** [review]

**Comment by jm-trd-ms** [comment]
#sign-off offline and merged.

**Pull request merged**